requirements.txt

```diff
@@ -6,4 +6,6 @@ pytest
 types-requests
 types-cachetools
 types-python-jose
+skyfield==1.48
+swifttools==3.0.21
```

Comment:
Why are these pinned?

Reply:
I figured as they were used for tests I should pin them incase a future update of either breaks their output.

Reply:
I'd unpin them; if they break in the future then we can start restricting them.